### PR TITLE
515 seedfile update

### PR DESCRIPTION
**Pull request by jasonallenphotography**
I rule and have completed some work on Case Manager that's ready for review!
(I love that prepopulated first sentence! LOL)

This pull request updates the seedfile to resolve issues #515 and #586. Build passes test suite locally.

This pull request makes the following changes:
* One patient should have other contact information (name, phone, and relationship)
* One patient should have an appointment a week from the current date and have a clinic selected
* One patient should have a pledge submitted
* One patient should be marked resolved without DCAF
* One patient should have special circumstances
* All patients except Patients 0 and 1 have a note. Even-numbered patients have two notes.
* Patients have names that reflect their test case (e.g. 'Patient with a pledge submitted - 3' rather than previous 'Patient 3')

It relates to the following issue #s: 
* Satisfies #515
* Satisfies #586

### CODE REVIEW

**Comment by jasonallenphotography** [comment]
Also, shout-out to @gabezurita for his help with my prior Docker issues and learning Mongo's quirks!

File: db/seeds.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 Patient.destroy_all
 User.destroy_all
+Clinic.destroy_all
```
Comment (colinxfleming) [inline db/seeds.rb]: IIRC Clinic data is embedded in Patient, so we don't need to specifically destroy, right?

File: db/seeds.rb [hunk 2]
```diff
@@ -56,21 +57,61 @@
   end
 end
 
-# Add a note to Patient 2
+# Patients should have names that reflect what we're looking for (e.g. 'Patient With Other Contact' rather than 'Patient 2')
+# Add example of patient with other contact info
+  patient1 = Patient.find_by( name: "Patient 1")
```
Comment (colinxfleming) [inline db/seeds.rb]: This is dope stuff -- going to be SUPER useful for reviewing test cases. 

What are your thoughts about moving some of this closer to line 22, on actual point of patient creation? Worth the trouble, or probably nobody will ever really dig into this so whatever?

File: db/seeds.rb [hunk 3]
```diff
@@ -56,21 +57,61 @@
...
+  patient4.save!  
+
+# All patients except one or two should have notes
 note_text = 'This is a note ' * 10
-Patient.find_by(name: 'Patient 2')
-       .notes.create! full_text: note_text,
-                      created_by: user2
+additional_note_text = 'Additional note ' * 10
+Patient.all.each do |patient|
+  unless patient.name == "Patient 0" || patient.name == "Patient 1 with Other Contact info"
```
Comment (colinxfleming) [inline db/seeds.rb]: Patient 1 has a slightly different name than what's here, can you resolve?

File: db/seeds.rb [hunk 4]
```diff
@@ -56,21 +57,61 @@
...
+
+# Add example of patient with appointment one week from today && clinic selected
+  patient2 = Patient.find_by( name: "Patient 2")
+  patient2.update_attributes( name: "Patient with clinic and appointment(1wk) - 2",clinic_name: "Sample Clinic 1", appointment_date: Time.now + (7*24*60*60) )
+  patient2.save!
+
+# # Add example of patient with a pledge submitted
+  patient3 = Patient.find_by( name: "Patient 3")
+  pledge = patient3.pledges.new()
+    pledge.created_by = user2
```
Comment (colinxfleming) [inline db/seeds.rb]: Not my proudest moment as a PM, but we wound up running out of time before we could really get this part of the object model implemented cleanly. We're using the pregnancy model for this right now, can we move this over to utilize that?

**Pull request merged**